### PR TITLE
Migrate nft transfers

### PR DESCRIPTION
**Pull request by jkylling**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by jkylling** [comment]
Data quality:
```
different checksums for spells_prod.nft.transfers and dbt_jonas14_nft.transfers
 different checksum for column 15 (('unique_transfer_id', 'varchar')): b'\x088J\xf6pf\x83[' != b'\x94\x7f^\x81\xfc\xd2\x8f\xea'
valid=0, invalid=1, total=1
```
The invalid checksum should be caused by the goerli transfer id changing from using the prefix gnogoerlis to goerli.

**Comment by jeff-dude** [comment]
waiting to merge until prod env(s) are ready after migrating/running some spark modifications

File: models/labels/addresses/__single_category_labels__/likely_bots/optimism/labels_optimism_likely_bot_addresses.sql [hunk 1]
```diff
@@ -21,10 +21,10 @@ WITH sender_transfer_rates AS (
             , SUM(CASE WHEN to IN (SELECT address FROM {{ ref('labels_optimism_likely_bot_contracts') }} WHERE name != 'chain ops bot') THEN 1 ELSE 0 END) AS bot_concentration_txs
             
             , SUM(CASE WHEN EXISTS (SELECT 1 FROM {{ source('erc20_optimism','evt_Transfer') }} r WHERE t.hash = r.evt_tx_hash AND t.block_number = r.evt_block_number) THEN 1 ELSE 0 END) AS num_erc20_tfer_txs
-            , SUM(CASE WHEN EXISTS (SELECT 1 FROM {{ ref('nft_transfers') }} r WHERE t.hash = r.tx_hash AND t.block_number = r.block_number AND blockchain = 'optimism') THEN 1 ELSE 0 END) AS num_nft_tfer_txs
+            , SUM(CASE WHEN EXISTS (SELECT 1 FROM {{ ref('nft_transfers_legacy') }} r WHERE t.hash = r.tx_hash AND t.block_number = r.block_number AND blockchain = 'optimism') THEN 1 ELSE 0 END) AS num_nft_tfer_txs
```
Comment (Hosuke) [inline models/labels/addresses/__single_category_labels__/likely_bots/optimism/labels_optimism_likely_bot_addresses.sql:24]: I am still confused with when to use `ref('\*_legacy')`, in legacy models or in dune sql models.

Comment (jeff-dude) [reply models/labels/addresses/__single_category_labels__/likely_bots/optimism/labels_optimism_likely_bot_addresses.sql:24]: just clarifying that we spoke in dm's about this, resolving now

**Pull request merged**